test/cases/enum.ts

```diff
@@ -20,10 +21,42 @@ export var schema = {
   "additionalProperties": false
 }
 
-export var types = `interface Enum {
+export var configurations = [
+  {
+    settings: {
+      useTypescriptEnums: false
+    },
+    types: `interface Enum {
   foo: "a" | "b" | "c";
   bar: number;
   baz: {
     a: number;
   };
 }`
+  }, 
+  {
+    settings: {
+      useTypescriptEnums: true
+    },
+    types: `enum Foo {
+  a,
+  b,
+  c
+}
+enum Bar {
+  One = 1,
+  Two = 2,
+  Three = 3
+}
+enum Baz {
+  [object Object],
+  [object Object],
+  [object Object]
+}
```

Comment:
As you can see here the expected output above won't compile for this particular case.  The enum members from the schema are

`enum: [{"a":1}, {"a":2}, {"a":3}]` and frankly I have no idea how I ought to render that for this specific case, let alone the generalized case.